bootstrap.sh

```diff
@@ -180,16 +237,15 @@ if [ -d appscale/.appscale/certs ]; then
                 fi
         fi
 
-        # This is an upgrade, so let's make sure we use a tag that has
-        # been passed, or the last one available. Let's fetch all the
-        # available tags first.
-        (cd appscale; git fetch --all)
-        (cd appscale-tools; git fetch --all)
-        if [ -z "$GIT_TAG" -o "$GIT_TAG" = "last" ]; then
-                GIT_TAG="$(cd appscale; git tag|tail -n 1)"
+
+        # Let's upgrade the repository: if GIT_TAG is empty we are on HEAD.
+        if [ -n "${GIT_TAG}" ]; then
+                (cd appscale; git checkout "$GIT_TAG")
+                (cd appscale-tools; git checkout "$GIT_TAG")
+        else
+                (cd appscale; git pull)
```

Comment:
I think we can just do `git checkout "${CURRENT_BRANCH}` here, right? We don't need to re-check origin at this point.